.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
```

Comment:
This looks like you may have borrowed this from somewhere on the internet? Is that right? If so, can you leave an html comment at the top that has the URL from where you borrowed this?